pages/contribution-flow.js

```diff
@@ -106,10 +108,15 @@ class NewContributionFlowPage extends React.Component {
     loadingLoggedInUser: PropTypes.bool,
     tags: PropTypes.arrayOf(PropTypes.string),
     step: PropTypes.oneOf(Object.values(STEPS)),
+    router: PropTypes.object,
   };
 
   componentDidMount() {
     this.loadExternalScripts();
+    const { router, data } = this.props;
+    const account = data?.account;
+    const path = router.asPath;
+    addParentToURLIfMissing(router, account, path.replace(`/${account?.slug}`, ''));
```

Comment:
```suggestion
    addParentToURLIfMissing(router, account, path.replace(new RegExp(`^/${account?.slug}/`), ''));
```

Reply:
Thanks. I've added this one; except a small change; the final `/` sign should be removed as in `addParentToURLIfMissing` we are not accounting for that. So,

```
addParentToURLIfMissing(router, account, path.replace(new RegExp(`^/${account?.slug}`), ''));
```

Reply:
Then rather do:

```es6
addParentToURLIfMissing(router, account, path.replace(new RegExp(`^/${account?.slug}/`), '/'));
```

The reason why I'm pushing for this change is to prevent replacing parts of the string that are not meant to be replaced. It might not be useful now, but it doesn't cost much to make this future-proof.

Reply:
Thanks. I've changed that. 👍🏽